MainApp/templates/MainApp/profile.html

```diff
@@ -102,7 +102,7 @@ <h2>{{account.user.first_name}} {{account.user.last_name}}</h2>
         <h3 style="color: rgb(99, 99, 99);">@{{account.user.username}}</h3>
       </a>
       <div style="width: 30%; float: right;"> 
-        <form method="post" action="{% url 'MainApp:followerstab' account.following.username %}">
+        <form method="post" action="{% url 'MainApp:followerstab' user %}">
```

Comment:
Calling the correct View with the authenticated username...old version was redirecting the the 'followed' user's profile page